packages/render/src/render.js

```diff
@@ -8,6 +8,9 @@ const renderImage = (ctx, node) => {
   const {style, options, box } = node; 
   ctx.addImage({...options,...style,...box});
 };
+const renderMedia = (ctx, node) => {
+  ctx.addMedia(node.props);
```

Comment:
You can see how other functions deal with the parameters, and copy it.
the the idea is that `style` and `box` are properties for positioning and sizing, and `options` contains all other props.
`options` is supposed to be created in `styleMedia` function

```
const {style, options, box } = node; 

ctx.addMedia({...options,...style,...box});
```
apart from this, great job, you properly get how to create newprimitive.